src/views/collection-view/components/collection-meta/index.tsx

```diff
@@ -6,30 +6,24 @@ import Heading from 'components/heading'
 import IconTile from 'components/icon-tile'
 import InlineLink from 'components/inline-link'
 import Text from 'components/text'
+import { Collection } from 'lib/learn-client/types'
 import s from './collection-meta.module.css'
+import CollectionMetaProgress from '../collection-meta-progress'
 
 interface CollectionMetaProps {
+	collection: Collection
```

Comment:
We now pass in the full `collection` data as a straightforward way to get what we need to `CollectionMetaProgress`